### PR TITLE
feat: as-1903; session management for LPA questionnaire

### PR DESCRIPTION
**Pull request by danWatkinson**
## Ticket Number
https://pins-ds.atlassian.net/browse/AS-1903

## Description of change
pinning down the session behaviour for LPA questionnaire

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [x] My commit history in this PR is linear
- [x] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: lpa-submissions-e2e-tests/cypress.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "demoDelay": 0,
     "DOCUMENT_SERVICE_BASE_URL": "http://localhost:3001/api/v1",
     "ASSUME_LIMITED_ACCESS": false,
-    "APPEAL_REPLY_SERVICE_BASE_URL": "http://localhost:3002"
+    "APPEAL_REPLY_SERVICE_BASE_URL": "http://localhost:3002",
```
Comment (danWatkinson) [inline lpa-submissions-e2e-tests/cypress.json:11]: mental note - agree these should have the /api/v1 bit baked in at config //todo

File: lpa-submissions-e2e-tests/cypress/fixtures/anAppeal.json [hunk 2]
```diff
@@ -0,0 +1,93 @@
+{
```
Comment (danWatkinson) [inline lpa-submissions-e2e-tests/cypress/fixtures/anAppeal.json:1]: stole an arbitrary appeal from team1's e2e tests; there would likely be an advantage to testing more than one appeal type at some stage...

Comment (davehlewis) [reply lpa-submissions-e2e-tests/cypress/fixtures/anAppeal.json:1]: for consistency this the data that is seeded: https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/feature/as-1903-lpa-sessions/data/data/appeals-service-api/001-appeals.json

Comment (danWatkinson) [reply lpa-submissions-e2e-tests/cypress/fixtures/anAppeal.json:1]: in all honesty we should have a bunch of these that we feed in to different tests to triangulate with.. if you check the assertions somewhere down below against the 'appeal sidebar' we're checking that the value displayed is the value we put in, so if we can't just swap appeals in and out we're presumably finding bugs?

**Comment by danWatkinson** [comment]
binned this off for a new branch to 1) rebase from master, and 2) bring my PR up towards the top of the pile because it was 2 weeks old and looked like it had been forgotten
->https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/1053

**Pull request closed without merging**